app.js

```diff
@@ -4,7 +4,9 @@ const app = express();
 const environment = process.env.NODE_ENV || 'development';
 const configuration = require('./knexfile')[environment];
 const database = require('knex')(configuration);
-const cors = require('cors')
+const cors = require('cors');
+
+app.use(express.json());
 
```

Comment:
I'm not sure why/how I didn't have this line in mine.   Everything seems to be passing still.